### ISSUE TITLE
α-BN Results 

### ISSUE THREAD

**Issue by tuwangwenx** [kiwi12138/RealisticTTA#1]
In Table 1, the batch size is 4, 2, 1. The results I ran are completely different from yours. Could you give the forward_and_adapt function you have here. I directly used the code given by Online Test-time Adaptation. I also modified self.norm = nn.BatchNorm2d(self.layer.num_features, affine=False, momentum=cfg.TEST.MOMENTUM, track_running_stats=True) in AlphaBatchNorm.
![image](https://github.com/kiwi12138/RealisticTTA/assets/95834252/8f855cbf-56a2-4ca8-98db-ebf0481e5e5e)

**Comment by kiwi12138** [comment]
If you would like to reproduce the Alpha_BN results, you can directly follow the original code of Online Test-time Adaptation[https://github.com/mariodoebler/test-time-adaptation/tree/main](url). Just modify the batchsize parameter in norm_alpha.yaml and run.  You don't need to modify any other functions.

**Comment by tuwangwenx** [comment]
> If you would like to reproduce the Alpha_BN results, you can directly follow the original code of Online Test-time Adaptation[https://github.com/mariodoebler/test-time-adaptation/tree/main](url). Just modify the batchsize parameter in norm_alpha.yaml and run. You don't need to modify any other functions.

Thank you for your reply and your paper. Could you please provide me with the AdaptBN code?
